clang/lib/Sema/SemaExpr.cpp

```diff
@@ -62,7 +63,42 @@
 
 using namespace clang;
 using namespace sema;
+class GetSymbolicName {
+private:  
+std::string LastString;
```

Comment:
I suggest you give a better name for this variable. Why is it called `LastString`?